package.json

```diff
@@ -31,14 +31,14 @@
     "format:md": "prettier --write '**/*.md'",
     "format:ts": "prettier --write '**/*.ts{,x}'",
     "lint": "eslint src test --ext .ts --fix && yarn format",
-    "test": "yarn build && yarn lint && jest && yarn clean",
+    "test": "jest",
     "watch": "jest --watch",
     "watch:debug": "yarn watch --runInBand --verbose",
     "coverage": "jest --coverage",
     "ci:prep": "yarn build && yarn ci:lint",
     "ci:lint": "eslint src test --ext .ts && yarn format",
     "ci:test:vanilla": "jest ./test/vanilla",
-    "ci:test": "yarn ci:prep && yarn ci:test:vanilla && yarn clean",
+    "ci:test": "yarn ci:prep && yarn test && yarn clean",
```

Comment:
I think we may want to consider adding [parallelism or using a workflow to run these at the same time](https://github.com/infinitered/ignite/issues/2071#issuecomment-1233185541)